images/Dockerfile.ubuntu

```diff
@@ -11,10 +11,8 @@ RUN apt-get update \
   && add-apt-repository ppa:oibaf/test \
   && apt-get update \
   && apt-get install -y --no-install-recommends \
-    binutils \
     conntrack \
     console-data \
-    console-setup \
```

Comment:
I think we need `console-setup`, IIRC for displaying correctly the QR code: #212

Reply:
Based on the PR that fixed that issue, we need `console-data`, which this change keeps.

`console-setup` is not recommended by `console-data` either: https://packages.ubuntu.com/kinetic/console-data

Reply:
I can't test this to verify it because I can't seem to build isos locally.  I've put more details in #875.

Reply:
Just built a new image without `console-setup` and it shows the QR properly.